templates/definition/charger/go-e-v3.yaml

```diff
@@ -11,10 +11,16 @@ requirements:
       Benötigt mindestens Firmware 052.1 oder neuer.
 
       Für 1P/3P-Phasenumschaltung muss die HTTP API v2 im Charger aktiviert sein.
+
+      Bei manchen Fahrzeugen müssen in der App bei den Auto-Einstellungen Änderungen vorgenommen werden.
```

Comment:
Einfacher:

    Bei manchen Fahrzeugen kann es notwendig sein, in der Go-E App "Ausstecken simulieren" zu aktivieren.

Reply:
Was spricht dagegen dies immer zu aktivieren?

Reply:
@andig  hast recht, als ich den Draft erstellt habe, kannte ich den Inhalt der App noch nicht.

@premultiply  könnte es sein, dass das dann auch Probleme macht (ich hab kein go-e)

Reply:
Die meisten Boxen machen das immer vollautomatisch ohne das der Anwender dies merkt oder abschalten könnte.